ui/app/templates/components/json-viewer.hbs

```diff
@@ -1,11 +1,9 @@
-<IvyCodemirror
+<div
   data-test-json-viewer
-  @value={{this.jsonStr}}
-  @options={{hash
-    mode="javascript"
+  {{code-mirror
+    content=this.jsonStr
```

Comment:
`suggestion`:  If you invoke the `stringifyObject` helper here and pass `this.json` instead, we can delete the backing component class.